src/main/java/com/google/api/codegen/CommentPatterns.java

```diff
@@ -0,0 +1,14 @@
+package com.google.api.codegen;
+
+import java.util.regex.Pattern;
+
+/**
+ * Abstract base class for formatting source comments to follow language-idiomatic style.
```

Comment:
(FWIW: I had originally considered making the methods of `RDocCommentFixer` and `PythonSphinxCommentFixer` nonstatic and deriving them from an abstract base class, but I don't know that there's any benefit to doing so -- that might just be an unnecessary constraint.)